components/css/componentStyle/Alert.css

```diff
@@ -0,0 +1,59 @@
+.alert{
+    border-radius: 5px;
+    border: 1px solid black;
+    padding: 1.5rem 4rem 1.5rem 1.5rem;
+    margin-top: 1.5rem;
+    margin-bottom: 1.5rem;
+    position: relative;
+}
+
+.primary__alert{
+    background-color: #cfe2ff;
```

Comment:
you can variable and declare that color there and use that color name here, it will give you more flexibilty.